static/rumble/bar/index.html

```diff
@@ -5,9 +5,9 @@
     <link rel="icon" type="image/svg+xml" href="/inky-logo.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Visualization</title>
-    <script type="module" crossorigin src="../assets/bar.ab469926.js"></script>
-    <link rel="modulepreload" crossorigin href="../assets/RumbleLoader.c6d6c914.js">
-    <link rel="modulepreload" crossorigin href="../assets/Grid.be76391d.js">
+    <script type="module" crossorigin src="../assets/bar.ae60c53a.js"></script>
```

Comment:
I know we had relative links before, but can these be made to be absolute links, like in `static/rumble/blog/index.html` below?

Reply:
That's a good catch, I updated it to be `/rumble/assets/...` for these. I still don't quite know how to make the upstream `visualization-ui` React app use a subdirectory, but when I figure that out it will be easier to make these urls consistent.